net-fs/davfs2/davfs2-1.5.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools linux-mod user
+
+DESCRIPTION="Linux FUSE (or coda) driver that allows you to mount a WebDAV resource"
+HOMEPAGE="http://savannah.nongnu.org/projects/davfs2"
+SRC_URI="http://mirror.lihnidos.org/GNU/savannah/davfs2/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+RESTRICT="test"
+
+DEPEND="dev-libs/libxml2
+	net-libs/neon
+	sys-libs/zlib"
+RDEPEND="${DEPEND}"
+
+pkg_setup() {
+	enewgroup davfs2
+}
+
+src_prepare() {
+	eautoreconf
+}
+
+src_configure() {
+	econf dav_user=nobody --enable-largefile --docdir=/usr/share/doc/${P}
+}
+
+src_compile() {
+	emake
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+}
```

Comment:
Fixed. They overrode src_compile() because it was inheriting linux-mod. linux-mod has a default action for src_compile() which doesn't compile the source because this is not a kernel module.

It seems the change has been introduced 9 years ago and nobody asked the question you asked during all this time:
https://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/net-fs/davfs2/davfs2-0.2.8.ebuild?hideattic=0&r1=1.2&r2=1.3

It also doesn't need to inherit autotools because eautoreconf is not needed. fixed that too.